### DISCUSSION TITLE
Allow portfile.cmake to get information defined in CONTROL file

### DISCUSSION THREAD

**Discussion by tobbi** [microsoft/vcpkg#17782] [Ideas]
It would be great to get the version or other things defined in the CONTROL file to use as a reference to vcpkg_from_github.

Best would be something to get the major, minor and patch version number using vcpkg functions or variables.

**Comment by LilyWangL** [comment]
https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/control-files.md

**Comment by isanych** [comment]
Side effect of this PR - https://github.com/microsoft/vcpkg/pull/7129 - versions (and other info from control) are exposed in cmake
